src/main/java/org/smartregister/domain/FormField.java

```diff
@@ -0,0 +1,57 @@
+package org.smartregister.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+@JsonSerialize
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+public class FormField {
```

Comment:
This class should remain in core its not being used on this library

Reply:
Actually FormTest class is being referenced in ContactPointTest and UserTest class, therefore I had to move these classes too.

Reply:
Moved back to core project